src/illumidesk/spawners/spawners.py

```diff
@@ -1,27 +1,50 @@
 import os
-import shutil
 
 from dockerspawner import DockerSpawner
+from illumidesk.spawners.hooks import custom_auth_state_hook
```

Comment:
The black formatting tool doesn't add line spacing between modules but we should do that according to our style guide.